Src/libCZI/utilities.cpp

```diff
@@ -423,3 +424,158 @@ tString trimImpl(const tString& str, const tString& whitespace)
     LoHiBytePackStrided_C(ptrSrc, sizeSrc, width, height, stride, dest);
 }
 #endif
+
+void RectangleCoverageCalculator::AddRectangle(const libCZI::IntRect& rectangle)
+{
+    if (!rectangle.IsValid())
+    {
+        return;
+    }
+
+    if (this->splitters_.empty())
+    {
+        this->splitters_.push_back(rectangle);
+    }
+    else
+    {
+        for (auto i = this->splitters_.begin(); i != this->splitters_.end(); ++i)
```

Comment:
Could we use a more concrete iterator name instead of `i` which I would expect to be used for an index while scanning through

Reply:
renamed to "splitter"